front/app/components/Author/index.tsx

```diff
@@ -140,10 +141,12 @@ const Author = memo(
   }: Props) => {
     const locale = useLocale();
     const { data: author } = useUserById(authorId);
-    const authorCanModerate =
-      !isNilOrError(author) &&
+    const showModeratorStyles =
       showModeration &&
-      canModerateProject(projectId, { data: author.data });
```

Comment:
For now this has worked for proposals because canModerateProject first checks if the user is an admin (which happens to be the same check we do in canModerateInitiative). Making this a bit more explicit